starter-code/spotify.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Spotify</title>
+  <link rel="stylesheet" href="./css/spotify.css">
+</head>
+<body>
+<div class="yeezy-container">
+<div class-"inner-container">
+    <div class="yeezy-text">
+        <h1>It's as yeezy as Kanye West.</h1>
+        <hr>
+        <h2>Search</h2>
+        <p>Know what you want to listen to?</p>
+     <p>Just search and hit play.</p>
+        <h2>Browse</h2>
+     <p>Check out the latest charts,
+        brand new releases and great
+        playlists for right now. </p>
+        <h2>Discover</h2>
+        <p>Enjoy new music every Monday</p>
+       <p> with your own personal playlist.
+        Or sit back and enjoy Radio.</p>
+    </div>
+    <div class="kanye-img">
+        <img src="./images/spotify-app.jpg" alt="">
+
+    </div>
+    <div class="logo">
+        <img src="./images/spotify-icon-white.png" alt="">
```

Comment:
Maybe setting it as a background image with background-size will make its positioning a bit easier